packages/design-system/src/components/WIP/Accordion/Primitive/CollapsiblePanel.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { forwardRef, ReactChild, Ref, useState, useEffect } from 'react';
+import classNames from 'classnames';
+import CollapsiblePanelHeader from './CollapsiblePanelHeader';
+import { PanelHeaderAction } from './types';
+
+import styles from './CollapsiblePanel.module.scss';
+
+type CollapsiblePanelPropsType = {
+	a11yId: string;
+	children: ReactChild;
+	managed?: boolean;
+	expanded?: boolean;
+	index?: number;
+	title: string;
+	action?: PanelHeaderAction;
+	size?: 'S' | 'M';
+	metadata?: ReactChild[];
+	isFirst?: boolean;
+	isLast?: boolean;
+	disabled?: boolean;
+	onToggleExpanded?: (index: number) => void;
+};
```

Comment:
```suggestion
} & Omit<HTMLAttributes<HTMLDivElement>, 'className' | 'style'> & DataAttributes;
```

To enable `data-test-id` and whatever else on panels for tracking